fastai/tabular/learner.py

```diff
@@ -1,17 +1,21 @@
-# AUTOGENERATED! DO NOT EDIT! File to edit: nbs/43_tabular.learner.ipynb (unless otherwise specified).
+# AUTOGENERATED! DO NOT EDIT! File to edit: nbs/nbs/43_tabular.learner.ipynb (unless otherwise specified).
```

Comment:
Any idea why this would be showing nbs/nbs/?  I haven't ran across that before, but can help troubleshoot if needed

Reply:
Thank you @kevinbird15. I am not too sure about this. As pointed out by @dienhoa, this PR has 9 files changed and I am not sure if that has anything to do with this rendering.

I did make a mistake in that I executed `nbdev_clean_nbs` before `nbdev_build_lib` and `nbdev_build_doc`. Not sure if that resulted in this.

Reply:
@moarshy just run `nbdev_build_lib` from the root directory of the project, don't pass `--fname`